app/views/wiki_menu_items/edit.html.erb

```diff
@@ -57,7 +57,7 @@ See doc/COPYRIGHT.rdoc for more details.
       <%= form.check_box 'new_wiki_page' %>
       <%= form.label 'new_wiki_page', l(:label_wiki_show_new_page_link) %>
 
-      </br>
+      <br>
```

Comment:
@NobodysNightmare HTML 5 doesn't require XHTML syntax. I think it's safe to move away from this syntax, although there are some gotchas when it comes to Angular tag directives.

http://www.techrepublic.com/blog/10-things/10-things-you-should-know-about-html5/